src/config/config.ts

```diff
@@ -1,74 +1,58 @@
 import { parseLocations } from "../utils/itemUtils"
 
 export const appConfig = {
-  environment: process.env.APP_ENV || "production",
+  environment: process.env.ENV || "production",
```

Comment:
I think `APP_ENV` is less confusing than `ENV` but not a big deal.

Reply:
I agree. Can change it to APP_ENV, will just need to update the env variable name in vercel and aws.